src/test/java/io/vertx/ext/apex/handler/StaticHandlerTest.java

```diff
@@ -133,7 +133,9 @@ public void testContentHeadersSet() throws Exception {
       String contentType = res.headers().get("content-type");
       String contentLength = res.headers().get("content-length");
       assertEquals("application/json", contentType);
-      assertEquals(18, Integer.valueOf(contentLength).intValue());
+      // Account for CRLF if checked out on Windows
```

Comment:
Not sure that it's a good idea, but it is the default setting for Git and TortoiseGit on Windows.  So likely I won't be the last person to encounter this, even if I change my setup.